pkg/apis/serving/metadata_validation.go

```diff
@@ -84,6 +84,7 @@ func ValidateRolloutDurationAnnotation(annos map[string]string) (errs *apis.Fiel
 func validateKnativeAnnotations(annotations map[string]string) (errs *apis.FieldError) {
 	for key := range annotations {
 		if !allowedAnnotations.Has(key) && strings.HasPrefix(key, GroupNamePrefix) {
+			fmt.Println("#### failed for ", key)
```

Comment:
guessing you didn't mean to leave this in :D

Reply:
yeah :)